src/main/java/com/knu/buga1chuk/data_structure/FlexibleArray.java

```diff
@@ -93,7 +97,7 @@ public void print() {
                 output.append(", ");
             }
         }
-        System.out.println(output);
+        LOG.info(String.valueOf(output));
```

Comment:
It is possible to simply use 'toString()' method?

Reply:
fix